src/worker/helpers.go

```diff
@@ -208,6 +213,63 @@ func stopOL(_ string) error {
 	return fmt.Errorf("worker didn't stop after 60s")
 }
 
+func cleanupOL(olPath string) error {
```

Comment:
Let's name this something like dirtyShutdownCleanup so it's clear we wouldn't normally do this.